<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1258</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,9,2">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
       <item>
        <widget class="QPushButton" name="open_folder">
         <property name="text">
          <string>Open Folder</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="qlist_images">
         <property name="styleSheet">
          <string notr="true">background-color:white</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="layoutMode">
          <enum>QListView::Batched</enum>
         </property>
         <property name="batchSize">
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="qlabel_image">
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>AnnotationBox</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QListWidget" name="list_widget"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1258</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>file </string>
    </property>
   </widget>
   <addaction name="menufile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionzoom">
   <property name="text">
    <string>zoom </string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>open_folder</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>selectDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qlist_images</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>item_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>item_click()</slot>
  <slot>selectDir()</slot>
  <slot>onListItemClick()</slot>
 </slots>
</ui>
